zerver/webhooks/front/doc.md

```diff
@@ -0,0 +1,37 @@
+Front lets you manage all of your communication channels in one place,
+and helps your team collaborate around every message. Follow these steps
+to recieve Front notifications without leaving Zulip!
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+Then, log into your Front account, and do the following:
+
+1. Go to the **Settings** of your organization.
+   ![](/static/images/integrations/front/001.png)
+
+2. Enable webhook notifications.
+   ![](/static/images/integrations/front/002.png)
+   ![](/static/images/integrations/front/003.png)
+
+3. Pick the types of events for which you want to receive notifications,
+and set the URL you construsted for the Front bot as a target webhook.
+   ![](/static/images/integrations/front/004.png)
+   ![](/static/images/integrations/front/005.png)
+
+Finally, go to the **Settings** of your Zulip organization, and add a new
+filter on the **Filter settings** page.
+
+**Pattern**: `cnv_(?P<id>[0-9a-z]+)`.
+**URL format string**: `https://app.frontapp.com/open/cnv_%(id)s`.
+
+![](/static/images/integrations/front/006.png)
+
+This step is necessary to set a link to a Front conversation as a name of
+the corresponding topic in Zulip.
+
+{!congrats.md!}
+
+![](/static/images/integrations/front/007.png)
+![](/static/images/integrations/front/008.png)
```

Comment:
We do want these last screenshots that depict sample messages in Zulip. But they don't need to be as large as they are in this PR. One screenshot with two example messages should be fine! :)